src/Containers/Artist/style.module.scss

```diff
@@ -4,7 +4,7 @@
 .container {
   margin-bottom: 7vh;
   @include portrait {
-    margin-bottom: mb();
+    margin-bottom: calc(100vw/11);
```

Comment:
@lmccart Lauren is this ok?
Fixing the following:
![Screenshot_20200528_171354](https://user-images.githubusercontent.com/11205091/83206905-caa66680-a106-11ea-9634-cfa31ec02403.png)

to:
![Screenshot_20200528_171548](https://user-images.githubusercontent.com/11205091/83206941-e6117180-a106-11ea-89ea-484ece9a2ebc.png)